salt/_states/caasp_etcd.py

```diff
@@ -27,7 +27,9 @@ def etcdctl(name, retry={}, **kwargs):
               'until': None}
     retry_.update(retry)
 
-    args = __salt__['caasp_etcd.get_etcdctl_args_str']()
+    skip_this = kwargs.pop('skip_this', False)
```

Comment:
General question: why bury this kind of arguments inside `kwargs` inside of adding them as regular arguments with defaults?

Reply:
Maybe not in this case in particular, but this kind of arguments are passed from function to function without being processed (and they are usually hidden/obscure arguments, so it is not very important)

Reply:
I think where possible we should avoid this style (except when you need to forward unknown arguments to an inner function), this makes the code harder to read IMO (you need to read the body in order to find out what arguments it can take).

Reply:
I agree with @ereslibre, but this is just my personal taste.

Reply:
This is a really subjective thing. In this case, most `**kwargs` go straigth to `caasp_cmd.run` (so we could use, for example, the `onlyif` argument when using this `caasp_etcd.etcdctl` state).

We must also extract some arguments used for for building the list of `etcdctl` arguments. `skip_if` is currently the only argument extracted. Adding it expllicitly in the arguments of this function would, in my view, give it more importance than what it really has, but as I said, this is very subjective.

However, as @flavio said, it should be documented in some way, so I will add some comment...

Reply:
Done.